src/asm/rpn.c

```diff
@@ -222,7 +223,8 @@ void rpn_CheckHRAM(struct Expression *expr, const struct Expression *src)
 		/* That range is valid, but only keep the lower byte */
 		expr->nVal &= 0xFF;
 	} else if (expr->nVal < 0 || expr->nVal > 0xFF) {
-		yyerror("Source address $%x not in $FF00 to $FFFF", expr->nVal);
+		yyerror("Source address $%" PRIx32 " not in $FF00 to $FFFF",
```

Comment:
Might as well take the chance to fix the message here: "not _between_ $FF00 _and_ $FFFF".